packages/grid/_modules_/grid/models/api/gridColumnApi.ts

```diff
@@ -49,9 +49,9 @@ export interface GridColumnApi {
   /**
    * Allows to toggle a column.
    * @param field
-   * @param forceHide Optional value, if not provided the column will toggle.
+   * @param hide
    */
-  toggleColumn: (field: string, forceHide?: boolean) => void;
+  setColumnVisibility: (field: string, hide: boolean) => void;
```

Comment:
`setColumnVisibility: (field: string, hide: boolean) => void;` 
the hide arg is opposite to the name of the function therefore , I would rename `hide` to `isVisible` 
`setColumnVisibility: (field: string, true) ` => makes the column visible not hidden!!

Reply:
Yes, I'm also not a fan of `hide` but wanted to keep it as it is to see if it would make sense to change. There are benefits of using `hide` the main one is that it's needed to invert the value.

On another note - I'm starting to think that a boolean as a second param may not be the best. Maybe we can think of a different name for the API method?

Reply:
open to suggestions...

But for me, what you have done is good to go.
Should you add a test for the workflow?

Reply:
Nice. I was thinking of a test but there are already tests for hiding and showing columns. I didn't add new functionality.